tests/integration/models/reddit/test_collections.py

```diff
@@ -138,6 +138,16 @@ def test_update_description(self, _):
             collection.mod.update_description(new_description)
             assert new_description == collection.description
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_update_display_layout(self, _):
+        self.reddit.read_only = False
+        uuid = self.NONEMPTY_REAL_UUID
+        new_display_layout = "GALLERY"
+        with self.use_cassette():
+            collection = self.subreddit.collections(uuid)
+            collection.mod.update_display_layout(new_display_layout)
+            assert new_display_layout == collection.display_layout
+
```

Comment:
We need a few more tests:

1. One for each layout
2. One of the layouts in lower case
3. `""`
4. `"something random"`

Reply:
Added them, but my tests still won't pass because of the 401 error. I deleted the cassette as you said before rerunning the test, but that still didn't work. I'm unsure what I'm doing wrong. 

Just to clarify, I have 2 questions:
1. Do I need an access token?
2. When I set the environment variable for the test subreddit, does it need to be subreddit I have mod permissions over or do I just stick with "reddit_api_test"?

Reply:
1. Yes. For Windows you will need to use `setx` with the `/m` flag and set the following:
  - `setx prawtest_client_id myclientid`
  - `setx prawtest_client_secret myclientsecret`
  - `setx prawtest_user_agent praw_pytest`
  - `setx prawtest_test_subreddit a_sub_you_mod`
  If you are using password authentication, then set the following:
  - `setx prawtest_username myusername /m`
  - `setx prawtest_password mypassword /m`
  If you are using a refresh token, then set:
  - `setx prawtest_refresh_token refresh_token /m`
2. Yes, you must be a mod of the test subreddit.

Reply:
It worked! I'm not sure why the way I did it (edited the env variables in Control Panel) had no effect though.